dluhc-component-library/src/components/siteSelectionForm/SiteSelectionForm.tsx

```diff
@@ -32,10 +33,16 @@ const createValidationSchema = (key: string, formSchema: FormPageSchema) => {
     case "number":
       validationShape = number();
       break;
+    case "array":
+      validationShape = array().of(string());
```

Comment:
Just realised `string()` part should come from `property?.items?.type` so we will want to pull this switch statement out in some way. Happy to have a chat about it